topics/whats-new-in-kotlin-for-kmm.md

```diff
@@ -0,0 +1,32 @@
+[//]: # (title: What's new in Kotlin for KMM)
+[//]: # (auxiliary-id: Whats_new_in_Kotlin_for_KMM)
+
+KMM is part of the larger Kotlin ecosystem and leverages Kotlin features and improvements for a better mobile developer experience. 
+[Every Kotlin release](https://kotlinlang.org/releases.html#release-details) brings features and improvements that are helpful for mobile developers like you. 
+
+Here you can find a short summary of the features Kotlin provides for developing multiplatform mobile applications.
+
+## Kotlin 1.4.20 for KMM
+
+[Kotlin 1.4.20](https://kotlinlang.org/docs/reference/whatsnew1420.html) introduces a number of features, improvements, and bug fixes that are helpful for KMM:
+
+* **CocoaPods plugin improvements**:
+    * Rebuilding dependencies only when necessary.
+    * Ability to add dependencies on libraries from a custom spec repository, Git repository, or archive, as well as on libraries with custom cinterop options.  
+      Learn more about [adding dependencies](add-dependencies.md#with-cocoapods) and [these improvements](https://kotlinlang.org/docs/reference/whatsnew1420.html#cocoapods-plugin-improvements).
```

Comment:
Let's maybe mention CocoaPods explicitly in this sentence? 
For example `Learn more about [adding CocoaPods dependencies]`

Reply:
Done